docs/source/sample_scripts/get_task_logs.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+from genestack_client import TaskLogViewer, make_connection_parser, get_connection
+
+# add extra arguments to the Genestack arguments parser for this script
+parser = make_connection_parser()
+parser.add_argument('--follow', action='store_true', help='Follow the logs\' output if the task is not done')
```

Comment:
Never escape quotes in python string, just select other quotes to define string
`"Follow the logs' output if the task is not done"`

Unnecessary escaping can lead to mistakes, like in that case.